samples/net/wpan_serial/prj.conf

```diff
@@ -25,3 +24,6 @@ CONFIG_SYS_LOG_USB_DEVICE_LEVEL=1
 
 CONFIG_ENTROPY_GENERATOR=y
 CONFIG_TEST_RANDOM_GENERATOR=y
+
+CONFIG_NET_APP_SETTINGS=y
+CONFIG_NET_APP_AUTO_INIT=n
```

Comment:
This is very strange option to set. IMHO, no sample should have this option set. If it's needed, the commit message should explain why.

Reply:
We initialize everything when border router application is connected through ttyACM.

Reply:
Following https://github.com/zephyrproject-rtos/zephyr/pull/9391, these should be renamed: CONFIG_NET_APP_* => CONFIG_NET_CONFIG_* (not all NET_APP_* of course, only those related to config).